libc/cmake/modules/LLVMLibCTestRules.cmake

```diff
@@ -68,6 +68,9 @@ function(get_object_files_for_test result skipped_entrypoints_list)
         endif()
         get_target_property(object_file_raw ${dep} "OBJECT_FILE_RAW")
         if(object_file_raw)
+          if(fq_target_name STREQUAL "libc.test.include.issignaling_c_test.__unit__" OR fq_target_name STREQUAL "libc.test.include.iscanonical_c_test.__unit__")
```

Comment:
should I add some comment here ?

Reply:
I'm trying to see if we have any better way than this without major changes.

Reply:
Can you add a TODO here to remove this special treatment later?  Thanks,